common/config/rush/common-versions.json

```diff
@@ -51,6 +51,7 @@
    * This design avoids unnecessary churn in this file.
    */
   "allowedAlternativeVersions": {
+    "@rushstack/rush-sdk": ["5.62.4", "^5.139.0"]
```

Comment:
Not a big deal, but normally we specify semver a little bit stricter. i.e we prefer exact version than `~` and prefer `~` than `^`